client/src/components/TopFeaturedAds.jsx

```diff
@@ -6,18 +6,23 @@ import AdAPI from "../apis/AdAPI";
 
 const TopFeaturedAds = (props) => {
   const [ads, setAds] = useState([]);
-  const [ad, setAd] = useState({ ad_image: ad1000, type: "1000x300", ad_link: "/"});
+  //const [ad, setAd] = useState({ ad_image: ad1000, type: "1000x300", ad_link: "/"});
 useEffect(() => {
     // Define a function fetchData that calls APIs which is then called in useEffect
     const fetchData = async () => {
       try {
         const response = await AdAPI.post("/getAdsBySize", { size: "1000x300"});
```

Comment:
```js
	//const [ad, setAd] = useState({ ad_image: ad1000, type: "1000x300", ad_link: "/"});
	useEffect(() => {
		AdAPI.post('/getAdsBySize', { size: '1000x300' })
			.then(response => setAds(response.data.data))
	}, [])
```
Use a promise instead of await

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise